.github/CODEOWNERS

```
@@ -301,7 +301,7 @@ datadog_checks_base/datadog_checks/base/checks/windows/              @DataDog/wi
 /palo_alto_cortex_xdr/assets/logs/                                   @DataDog/saas-integrations @DataDog/documentation @DataDog/logs-backend @DataDog/logs-core
 
 /snowflake/                                                          @DataDog/saas-integrations
-/snowflake/*.md                                                      @DataDog/saas-integrations @DataDog/documentation
+/snowflake/README.md                                                 @DataDog/saas-integrations @DataDog/documentation
```

Agent integrations or SAAS integrations? Or did you mean to change it to:

```suggestion
/snowflake/README.md                                                 @DataDog/saas-integrations @DataDog/documentation @DataDog/agent-integrations
```

---

Good question, I think you're onto something.

So it used to be that saas-ints exclusively owned all `.md` files in snowflake. What this change proposes is that saas-ints exclusively only own `README.md`. But they also own `snowflake` in general, so mb we need to say that agent ints owns changelog...

---

pushed a fix, @steveny91 lmk wyt.

---

Nice! Yeah I think the iteration should work. In my understanding originally, saas ints had exclusive ownership of the whole snowflake directory. Then you gave them exclusive ownership of the readme.md file in your initial commit. My understanding was that it didn't really change anything since they already had ownership of it.

---

That's exactly right, I didn't take into account that they owned the folder exclusively 🤦